core/modules/system/css/system.css

```diff
@@ -370,3 +370,86 @@ section,
 summary {
   display: block;
 }
+
+/**
+ * Special type of details elements, which use a customizable more/less toggle.
+ */
+details.more-less-toggle {
+  background-color: transparent;
+  border: none;
+  cursor: text;
+  margin: 0;
+  padding: 0;
+}
+
+/* Remove the default details toggle icon (triangle). */
+details.more-less-toggle > summary {
+  /* This must be combined with "display: none;" for ::marker below. */
+  list-style: none;
+  /* Prevent the summary text from triggering open/close of the details. */
+  pointer-events: none;
+}
+details.more-less-toggle summary::marker,
+details.more-less-toggle summary::-webkit-details-marker {
+  /* Required in order to remove the default toggle icon (see above). */
+  display: none;
+}
+
+details.more-less-toggle summary span.more-less-toggle-label {
+  padding: 0 3px;
+  /* Specifically trigger open/close of the details via the more/less toggle. */
+  pointer-events: auto;
+  /* Prevent the more/less text from being split from the toggle icon. */
+  white-space: nowrap;
+}
+/* Allow any links within the summary text to be clickable. */
+details.more-less-toggle summary a {
+  pointer-events: auto;
+}
+
+details.more-less-toggle summary > span.more-less-toggle-label::before,
+details.more-less-toggle summary > span.more-less-toggle-label::after {
+  color: #0074bd;
+  cursor: pointer;
+  padding: 0 2px;
+}
+details.more-less-toggle[open] summary .more-less-toggle-label::before {
+  content: attr(data-close-label);
+}
+details.more-less-toggle:not([open]) summary .more-less-toggle-label::before {
+  content: attr(data-open-label);
+}
+[dir="rtl"] details.more-less-toggle[open] summary .more-less-toggle-label::after {
+  content: attr(data-close-label);
+}
+[dir="rtl"] details.more-less-toggle:not([open]) summary .more-less-toggle-label::after {
+  content: attr(data-open-label);
+}
+
+/* The arrow icon next to the more/less toggle label. */
+details.more-less-toggle summary .more-less-toggle-label::after {
+  content: '▶';
+  display: inline-block;
+  font-size: 80%;
+  transition: 0.2s;
+}
+[dir="rtl"] details.more-less-toggle summary .more-less-toggle-label::before {
+  content: '◀';
+  display: inline-block;
+  font-size: 80%;
+  transition: 0.2s;
+}
+details.more-less-toggle[open] summary .more-less-toggle-label::after {
+  transform: rotate(90deg);
+}
+[dir="rtl"] details.more-less-toggle summary .more-less-toggle-label::after {
+  font-size: 100%;
+  transform: none;
+}
+[dir="rtl"] details.more-less-toggle[open] summary .more-less-toggle-label::before {
+  transform: rotate(-90deg);
+}
+
+details.more-less-toggle .details-content-wrapper {
+  margin-top: 0.5em;
+}
```

Comment:
```suggestion
}

/* Don't show the outline for details elements that have custom more/less labels. */
details summary.more-less-toggle {
  outline: none;
}
```

Reply:
@bugfolder I ended up fixing this with some `tabindex` properties set. See [`18c4d90` (#3629)](https://github.com/backdrop/backdrop/pull/3629/commits/18c4d90547a3322ed3d6fb1221dcd32e8e76c1ab)

Things seem to only work properly in Chrome though. See my recent comment in the issue.